app/app-config.gradle

```diff
@@ -15,23 +15,42 @@
  */
 
 ext {
-    // CI Build
-    isCiBuild = System.getenv("TRAVIS") == "true"
-
     // Freesound API
-    Properties apiProperties = loadFileProperties("$projectDir/freesound-api.properties")
-    freesound_api_key = apiProperties.getProperty("freesound.api.key", "null")
+    freesound_api_key = getFreesoundApiKey()
     freesound_api_url = "\"https://www.freesound.org/\""
 }
 
+def String getFreesoundApiKey() {
+    String freesoundApiKey = isCiBuild() ? getApiKeyFromEnv() : getApiKeyFromFile();
+    if (freesoundApiKey == null || freesoundApiKey.isEmpty()) {
+        throw new IllegalStateException(
+                "Could not find Freesound API key value in environment or property file")
+    }
+    return "\"$freesoundApiKey\""
+}
+
+private boolean isCiBuild() {
+    System.getenv("TRAVIS") == "true"
```

Comment:
That mostly why I've stuck with using a file to define the `freesound.api.key` property for developers, rather moving to all environment variables - platform independence and no differences when using in command-line Gradle vs Android Studio.